rules/tines_rules/tines_global_resource_destruction.yml

```diff
@@ -0,0 +1,41 @@
+AnalysisType: rule
+DisplayName: "Tines Global Resource Destruction"
+Enabled: true
+Filename: tines_global_resource_destruction.py
+Severity: Low
+Tests:
```

Comment:
please update test output in the pr description since it looks like you've added more cases than are reflected in the image

Reply:
It turns out we run `make test` in the actions on the PR, so me including the test results in the PR is actually a bad idea and as you pointed out, prone to errors.

Tests can be viewed here: https://github.com/panther-labs/panther-analysis/actions/runs/5270455765/jobs/9529981979?pr=786

Reply:
running make test for this particular rule id and including the output should not cause any errors, and i don't remember pointing any potential ones out

Reply:
please re run this command and include the output in the pr description so that we have record of title behavior and returned alert context for ease of visibility and inclusion to everyone viewing this pr

Reply:
I can put the output too if that saves time

```
Tines.Global.Resource.Destruction
	[PASS] Detection Trigger
		[PASS] [rule] true
		[PASS] [title] Tines [GlobalResourceDestruction] performed by [user@email.com] on [tines-log-source-name]
		[PASS] [dedup] Tines [GlobalResourceDestruction] performed by [user@email.com] on [tines-log-source-name]
		[PASS] [alertContext] {"actor": "user@email.com", "action": "GlobalResourceDestruction", "tenant_id": "1337", "user_email": "user@email.com", "user_id": "7331", "operation_name": "GlobalResourceDestruction", "request_ip": "98.224.225.84"}
	[PASS] Tines Login
		[PASS] [rule] false
```

Reply:
Wow Github isn't always updating for me, this is annoying! just saw your comments